### PR TITLE
close inbound compressions properly

### PR DESCRIPTION
**Pull request by patriknw**
- after quarantine we should never start new inbound compressions for
  that uid
- I noticed that the hit counting was creating new instances also for
  quarantined, and thereby scheduling advertisments

### CODE REVIEW

**Comment by patriknw** [comment]
I think that's a new failure. Created issue https://github.com/akka/akka/issues/21500
Probably unrelated to this PR

**Comment by patriknw** [comment]
PLS BUILD

File: akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala [hunk 1]
```diff
@@ -60,80 +60,117 @@ private[remote] final class InboundCompressionsImpl(
 
   private val stopped = new AtomicBoolean
 
-  private[this] val _actorRefsIns = new Long2ObjectHashMap[InboundActorRefCompression]()
-  private val createInboundActorRefsForOrigin = new LongFunction[InboundActorRefCompression] {
-    override def apply(originUid: Long): InboundActorRefCompression = {
+  // None is used as tombstone value after closed
+  // TOOD would be nice if we can cleanup the tombstones
```
Comment (ktoso) [inline akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala]: TOOD => TODO. Yeah we should likely cleanup somehow.

**Review by ktoso — APPROVED** [review]
LGTM

**Pull request merged**